### PR TITLE
DO-955 Add link to header logo

### PR DESCRIPTION
**Pull request by Litsyshyn**
Added redirect to https://dp.dataocean.us/auth/sign-in/ from landing header logo. https://razom.atlassian.net/browse/DO-955
It's not in task, but i guess it must be added also to footer logo.

Upd.
Figured out that reversal logo already available, so delete adde by myself and correct code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -452,8 +454,9 @@ <h1 class="section__title h1">
     <footer class="footer-wrapper">
         <div class="container">
             <div class="footer">
-                <img class="footer__item" src="static/img/logo-white.svg" class="header-logo">
-
+                <a class="footer__item" href="https://dp.dataocean.us/auth/sign-in/" target="_blank">
```
Comment (roman-tiukh) [inline index.html:457]: Use env variables for generate links

Comment (roman-tiukh) [reply index.html:457]: DO_FRONTEND_HOST

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (roman-tiukh) [inline index.html:457]: forgot to change here

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Litsyshyn) [reply index.html:457]: Yes, sorry.

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**